apps/web/modules/ui/components/dialog/index.test.tsx

```
@@ -0,0 +1,218 @@
+  test("DialogHeader handles dangerouslySetInnerHTML", () => {
+    const htmlContent = "<span>Dangerous HTML</span>";
+    render(<DialogHeader dangerouslySetInnerHTML={{ __html: htmlContent }} />);
+
+    const header = document.querySelector(".flex.flex-col");
+    expect(header).toBeInTheDocument();
+    expect(header?.innerHTML).toContain(htmlContent);
+  });
+
+  test("DialogFooter handles dangerouslySetInnerHTML", () => {
+    const htmlContent = "<span>Dangerous Footer HTML</span>";
+    render(<DialogFooter dangerouslySetInnerHTML={{ __html: htmlContent }} />);
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)